packages/es-components/src/components/containers/popover/Popover.js

```diff
@@ -239,7 +241,9 @@ Popover.propTypes = {
   /** Enable event handlers provided by Popper.js */
   enableEvents: PropTypes.bool,
   /** Sets the strategy for positioning the popover in Popper.js */
-  strategy: PropTypes.oneOf(['absolute', 'fixed'])
+  strategy: PropTypes.oneOf(['absolute', 'fixed']),
+  /** When using a React portal, such as sliding pane, this helps the arrow to stay aligned with the trigger */
+  enableKeepTogether: PropTypes.bool
```

Comment:
What happens if this is enabled when not using a React portal? Could this be something we just always have enabled behind the scenes without having to expose a new prop?

Reply:
From reading about it here https://popper.js.org/docs/v1/#modifiers..keepTogether it ended up having more to do with spacing, and the portal thing is really a nice benefit. I don't think we would want to turn it on all the time without more investigation into the side effects of doing that.

Reply:
From the docs it sounds like it would be fine, though there may be situations where we don't want it pointing exactly to the reference object. 🤷🏼‍♂️ Keeping the prop name the same as the popper prop would be my vote, since `keepTogether` is an action statement by itself.